kong/plugins/ldap-auth/asn1.lua

```diff
@@ -1,21 +1,51 @@
+require "resty.openssl.include.ossl_typ"
```

Comment:
This seems to be somekind of global thing, any other way?

Reply:
Ah, it seems to be running well after removing this line :)